ddp-workspace/projects/ddp-sdk/src/lib/services/activity/activityQuestionConverter.service.ts

```diff
@@ -210,4 +111,123 @@ export class ActivityQuestionConverter {
             return convertedGroup;
         });
     }
+
+    private initQuestionBuilders(): void {
+      this.questionBuilders = [
+          {
+              type: QuestionType.Boolean.toUpperCase(),
```

Comment:
If we are going to refactor, perhaps we should just rename the enum values to their uppercase values? The values we get from server are uppercase and does not help us to use them in lowercase. E.g.:
<img width="803" alt="Screen Shot 2021-02-08 at 11 19 03 AM" src="https://user-images.githubusercontent.com/29984380/107247648-7d9dc280-69ff-11eb-8378-f24b6be8a726.png">

Reply:
You are right.
Fixed in 36842b840f002035e924d49ec926a43804a46e98 (for blockType and questionType)